### DISCUSSION TITLE
[Loot Tracker] No option to "reset others"

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#15877] [Ideas]
-

**Comment by LRober09** [comment]
Working on it, let's see if the PR ever gets looked at :)

For the time being, it'll only support grouped loot (menu entry won't be visible for individual kills).

![plugin](https://user-images.githubusercontent.com/6581665/204053447-b555f9c0-9b07-4310-b118-eb0580586be1.png)

**Comment by LRober09** [comment]
PR posted: https://github.com/runelite/runelite/pull/15960
